squidDrone/tests/mock_libraries/imu/generic_imu.hpp

```diff
@@ -2,7 +2,7 @@
 #define SRC_GENERIC_IMU_HPP_
 
 #include <memory>
-#include "i2c_handler.hpp"
+#include "i2c.hpp"
```

Comment:
Is it a mock? If not use a mock

Reply:
Same as above. Original header is used, but implementation is mocked or will be mocked.